wagtail/admin/views/reports/aging_pages.py

```diff
@@ -68,7 +68,10 @@ def decorate_paginated_queryset(self, queryset):
         }
         for page in queryset:
             if page.last_published_by:
-                page.last_published_by_user = username_mapping[page.last_published_by]
+                page.last_published_by_user = username_mapping.get(
+                    page.last_published_by,
+                    _("user %(id)s (deleted)" % {"id": page.last_published_by}),
```

Comment:
```suggestion
                    _("User %(id)s (deleted)" % {"id": page.last_published_by}),
```

Bit of a nitpick but wondering if this should be `User 123 (deleted)` and not `user 123 (deleted)`. My reasoning is that the name would normally be a proper noun.

Reply:
Good point! We should also apply that to https://github.com/wagtail/wagtail/blob/05951b350196b059307d4c0233fdf773d4b41486/wagtail/models/audit_log.py#L227 and https://github.com/wagtail/wagtail/blob/05951b350196b059307d4c0233fdf773d4b41486/wagtail/models/audit_log.py#L236

Reply:
I don't have a preference on this, but to give a contrary perspective: "user" is not a proper noun, and what is displayed otherwise is the result of `get_user_model().get_username()`, which may return a name, an email address, etc., so depending on the project may not be a proper noun.

Reply:
I didn't realise we already had some similar code that uses lowercase 'user'. Sorry but maybe it's best to align with existing code for now. 

Good point Josh about the rabbit hole of proper noun usage.

Reply:
I'm not a native english speaker, but is this not a "capfirst"-issue?
The username or those fallbacks are used as standalone column values. We cannot use `capfirst()` on the column value, because of course username/email/uuid/... should be kept as is, but at least we could capitalize the first character of those fallback values ("System" and "User 123 (deleted)")?

If I look at the site history page, for me as a non-native speaker, "system" looks kind of wrong?!
![wagtail_site_history_system_capitalization](https://user-images.githubusercontent.com/468164/199008492-ff7cc2d1-ece7-48ef-ad94-5be4185675c4.png)

To avoid blocking this actual bug fix too long, maybe we could create a new improvement issue for this, of course only if this is an actual issue, you have to tell ;)

Reply:
You make a fair point - personally I would err on the side of not changing anything in this PR (unless a maintainer has a preference) - I've extracted a function for the user string so that can be updated in one place.